moduleframework/common.py

```diff
@@ -406,21 +417,11 @@ def getPackageList(self, profile=None):
         package_list = []
         if not profile:
             if 'packages' in self.config:
-                packages_rpm = self.config['packages'].get('rpms') if self.config[
-                    'packages'].get('rpms') else []
+                packages_rpm = self.config.get('packages',{}).get('rpms', [])
                 packages_profiles = []
-                for x in self.config['packages'].get('profiles') if self.config[
-                    'packages'].get('profiles') else []:
-                    packages_profiles = packages_profiles + \
-                                        self.getModulemdYamlconfig()['data']['profiles'][x]['rpms']
+                for profile_in_conf in self.config.get('packages',{}).get('profiles',[]):
```

Comment:
The same here as I have mentioned above.

Reply:
see above